### PR TITLE
Portofolio desktop version: Resposive Web Design

### PR DESCRIPTION
**Pull request by Marcraphael12**
check with the others if the media queries have been used correctly to switch from the mobile version to the desktop version!
I worked hard to make this, but maybe my work is not enough to deploy this project. So I would be happy to see your reviews and add the better solution you will propose!

### CODE REVIEW

**Review by vichuge — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @Marcraphael12, 

Good job so far!
Excellente project!!! I could see the hours working on this and check all in detail, I saw you with a great future as a front-end developer, you have it.

mario_ladders

## Good points 👍 
- No Linters error ✔️ 
- Your mobile design is... I ❤️ it, I'd pay for this, really! ✔️ 
- Seriously, your mobile design is perfect ✔️ 

## Aspects to improve ♻️

- [ ] The mobile design is perfect, but according to this project, you need to have tablet and desktop size too. I recommend you to check the figma template to know how should look on desktop and tablet, becasue at this point, the 3 are the same.
- This is the problem in desktop, when you see it, is the same as mobile but, it should be different
image

- [x] Something that could help for this is use media queries for the Desktop breakpoint in the CSS file.(min-width > 992px - 1100px)

```html
// Passes the requirement (propper size)
@media screen and (max-width: 1080px) {
/*
.
.

*/
}
```

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Marcraphael12** [comment]
> # Required Changes   
> Hi @Marcraphael12,
> 
> Good job so far!
> Excellente project!!! I could see the hours working on this and check all in detail, I saw you with a great future as a front-end developer, you have it.
> 
> mario_ladders
> 
> ## Good points 
> * No Linters error heavy_check_mark
> * Your mobile design is... I heart it, I'd pay for this, really! heavy_check_mark
> * Seriously, your mobile design is perfect heavy_check_mark
> 
> ## Aspects to improve 
> * [ ]  The mobile design is perfect, but according to this project, you need to have tablet and desktop size too. I recommend you to check the figma template to know how should look on desktop and tablet, becasue at this point, the 3 are the same.
> * This is the problem in desktop, when you see it, is the same as mobile but, it should be different
>   image
> * [x]  Something that could help for this is use media queries for the Desktop breakpoint in the CSS file.(min-width > 992px - 1100px)
> 
> ```
> // Passes the requirement (propper size)
> @media screen and (max-width: 1080px) {
> /*
> .
> .
> 
> */
> }
> ```
> 
> Cheers and Happy coding!
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Thank you a lot for this comment! I can see you opened the deployment link in the README file. But this deployment just hosts the mobile version of my project, for now, the desktop version has not yet been merged. To provide you a view of the last result of this work, please take a look at these pictures below!

Capture d’écran de 2021-06-17 08-54-07
Capture d’écran de 2021-06-17 08-54-43
Capture d’écran de 2021-06-17 08-54-48
Capture d’écran de 2021-06-17 08-54-54
Capture d’écran de 2021-06-17 08-55-14
Capture d’écran de 2021-06-17 08-55-18
Capture d’écran de 2021-06-17 08-55-23

**Comment by Marcraphael12** [comment]
> > # Required Changes
> > Hi @Marcraphael12,
> > Good job so far!
> > Excellente project!!! I could see the hours working on this and check all in detail, I saw you with a great future as a front-end developer, you have it.
> > mario_ladders
> > ## Good points
> > 
> > * No Linters error heavy_check_mark
> > * Your mobile design is... I heart it, I'd pay for this, really! heavy_check_mark
> > * Seriously, your mobile design is perfect heavy_check_mark
> > 
> > ## Aspects to improve
> > 
> > * [ ]   The mobile design is perfect, but according to this project, you need to have tablet and desktop size too. I recommend you to check the figma template to know how should look on desktop and tablet, becasue at this point, the 3 are the same.
> > * This is the problem in desktop, when you see it, is the same as mobile but, it should be different
> >   image
> > * [x]   Something that could help for this is use media queries for the Desktop breakpoint in the CSS file.(min-width > 992px - 1100px)
> > 
> > ```
> > // Passes the requirement (propper size)
> > @media screen and (max-width: 1080px) {
> > /*
> > .
> > .
> > 
> > */
> > }
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > Cheers and Happy coding!
> > Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
> > **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> > _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
> 
> Thank you a lot for this comment! I can see you opened the deployment link in the README file. But this deployment just hosts the mobile version of my project, for now, the desktop version has not yet been merged. To provide you a view of the last result of this work, please take a look at these pictures below!
> 
> Capture d’écran de 2021-06-17 08-54-07
> Capture d’écran de 2021-06-17 08-54-43
> Capture d’écran de 2021-06-17 08-54-48
> Capture d’écran de 2021-06-17 08-54-54
> Capture d’écran de 2021-06-17 08-55-14
> Capture d’écran de 2021-06-17 08-55-18
> Capture d’écran de 2021-06-17 08-55-23
Capture d’écran de 2021-06-17 09-06-34
I hope this is more descriptive!

File: index.html [hunk 1]
```diff
@@ -13,61 +13,71 @@
...
-
       <!--this section contain information about projects-->
       <section id="id-works-section">
+        
+        <ul class="class-project-container">
 
-        <!--Introduction to the the project list-->
-        <h1 class="class-sections-titles">Projects</h1>
+          <!--Introduction to the the project list-->
+          <li class="class-sections-titles works-section-title">Projects</li>
```
Comment (teekaytech) [inline index.html:59]: In the `Projects` section, kindly observe the following:

- [ ] The `Projects` text should read from top to bottom as observed in the Figma design.
- [ ] Next to the title taking 70% of the page's width, with no margin on the right, and aligned to the right there's a featured project.

_**Figma Design**_
figma

_**Your Design**_
design

_For other Regular (non-featured) projects_

- [ ] The first one is just a mockup image aligned to the left of the page with no margin on the page and it takes about 25% of the page's width.

- [ ] The second project shares the first one's row but is bigger in height (taking also part of the next row also) and looks just like the featured project but the description is above the picture; also takes about 25% of the page's width and is aligned completely to the right with no margin in the page.

- [ ] The third project is just a mockup image (like the first one) and is below the first project with no margin on top or left, this one takes 50% of the page's width.

- [ ] Lastly, there's a fourth project below the third one but starts in the center of the page and it takes about 30% of the page's width.

File: index.html [hunk 2]
```diff
@@ -156,21 +168,25 @@ <h2 class="class-project-name">Project name goes here</h2>
       <!--The section about my self-->
       <section id="id-section-about-my-self">
 
-        <!--Introduction to the the project list-->
-        <h1 class="class-sections-titles">About</h1>
+        <div class="about-container">
```
Comment (teekaytech) [inline index.html:171]: - [ ] Something seems to be wrong with the bullets of the list items listed in the `Languages`, `Skills`, and the `Frameworks` section: they seem to be overlapping on the text. I think you should have a look and fix it.
- [ ] You might want to consider increasing the margins to the left and right of all the components on your page. The components look so close to the left and right margins of the container compared to the Figma tempate at the moment.

**_Figma Template_**
figma

**_Your Design_**
design

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello team :wave:

**Thank you for stating the last reviewer's point of confusion.  Your PR  looks professional with a nice title and summary, including commit messages :+1:
Your project looks great so far 👏**

__However, you need to make some required changes before moving on to the next assignment.__
> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it :blush: 

### **Required Changes :recycle:**
Kindly check my inline comments below :bowing_man: 

### **Optional Changes** :zap: 
N/A

* * *
**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review unless it is requested otherwise.**

*Feel free to leave any questions or comments in the PR thread if something is not 100% clear.*

*REMINDER: As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can [request a second opinion using this form](https://airtable.com/shrQAqnBwek5a0O0s).*

_**Please make the required changes and submit another review request when you are done.**_

* * *

Re-reviewed by: [**Taofeek Olalere**](https://github.com/teekaytech)
*Cheers and Happy Coding* :smiley:

**Review by vichuge — APPROVED** [review]
# Approved project! ✅ 🥳 🎈 🎉🎉🎉🎉🎉🎉🎉🎉 

Hi @Marcraphael12, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

InbornCandidHornet-size_restricted

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**